### PR TITLE
Chef 13 Fixes

### PR DESCRIPTION
**Pull request by eldebrim**

### CODE REVIEW

File: test/integration/provisioning/node1.rb [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
+    nics: [{ net_id: ENV['OS_NETWORK_UUID'] }],
   },
                   ssh_username: 'centos',
                   ssh_options: {
-                    key_data: nil
+                    key_data: nil,
                   },
                   convergence_options: {
-                    chef_version: '12.18.31'
+                    chef_version: '12.18.31',
```
Comment (ramereth) [inline test/integration/provisioning/node1.rb]: Please update this to 13.8.5.

**Comment by eldebrim** [comment]
@ramereth There might be a bug in `fog-openstack` that causes it to append `v2.0/tokens` to any URL we pass. Do you think we should create a new environment variable that doesn't include the path?

**Comment by eldebrim** [comment]
[Chef 13: osl-ceph](https://trello.com/c/HYx6gTvf/972-chef-13-osl-ceph)

File: recipes/default.rb [hunk 2]
```diff
@@ -36,7 +36,8 @@
 
 delete_resource(:execute, 'change-ceph-conf-perm')
 
-directory '/etc/ceph' do
+edit_resource!(:directory, '/etc/ceph') do
```
Comment (ramereth) [inline recipes/default.rb]: I don't think you want ``!`` here.

**Comment by ramereth** [comment]
[Chef 13: Fix resource cloning with various roles](https://trello.com/c/YnEfltcl/923-chef-13-fix-resource-cloning-with-various-roles)

**Comment by ramereth** [comment]
@eldebrim I'm having trouble running the osd suite. It looks like we may need to ensure we don't run the newer version of the fog-openstack gem to fix it for now.

**Comment by eldebrim** [comment]
!bump major *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v2.0.0.  Have a nice day!

**Pull request merged**